Automattic/wp-calypso
# Jetpack Cloud: Update the Social icon.

#### Changes proposed in this Pull Request

The icon used in the sidebar was a placeholder. Now we have the final
design, this PR updates the SVG.

#### Testing instructions

* Load Jetpack Cloud from the live link
* Check that the icon for Social looks correct

<img width="247" alt="image" src="https://user-images.githubusercontent.com/96462/170131487-4dcaee1e-0b96-4935-b985-281c9d6e0a84.png">
